pkg/gatewayserver/gatewayserver.go

```diff
@@ -841,7 +832,7 @@ func (gs *GatewayServer) handleLocationUpdates(conn connectionEntry) {
 							"antennas",
 						},
 					},
-				}, callOpt)
+				}, gs.WithClusterAuth())
```

Comment:
I don't know if this is a good idea. Anyone can now spoof the location of any UDP gateway since we don't check for auth.